bears/generate_package.py

```diff
@@ -113,7 +117,18 @@ def main():
     else:
         bear_version = repr(bear_version) + '.' + str(int(time.time()))
 
-    for bear_file_name in sorted(set(glob('bears/**/*Bear.py'))):
+    local_files = sorted(set(glob('bears/**/*Bear.py')))
+
+    if args.bear:  # Checks if bear name entered is available on the machine
+        for f in local_files:
```

Comment:
so why exactly is this done?
local_files is a list of strings, right? why would you iterate through it? cant you just check whether each of the bears given as arg is in this list?
the only way i would see it the way youre doing it is so that you could give partial names like "coala" instead of "coalaBear", is this it? but wouldnt this give some false positives in some cases? like writing RBear and getting something like "Cpprbear" or so (cpprbear doesnt exist, im just making a scenario in which it would be bad)?